src/app/components/UserAssets/components/UserAssetsTableRow.tsx

```diff
@@ -147,13 +148,13 @@ export const UserAssetsTableRow: React.FC<IUserAssetsTableRow> = ({
                 interactionKind="hover"
                 content={
```

Comment:
The same issue as with `Receive`, pls make sure the button is vertically centered.
<img width="1244" alt="image" src="https://user-images.githubusercontent.com/14878713/224716868-fe7a2836-113f-49e4-8e47-d36151c6cf75.png">